UefiTestingPkg/AuditTests/PagingAudit/UEFI/AArch64/PagingAuditProcessor.c

```diff
@@ -13,6 +13,7 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 extern MEMORY_PROTECTION_DEBUG_PROTOCOL  *mMemoryProtectionProtocol;
 
 #define TT_ADDRESS_MASK  (0xFFFFFFFFFULL << 12)
+#define TT_AF            BIT10

```

Comment:
Usually, these bits would go into a common header file. Is that not done because such a file does is not clearly defined? Or you really prefer this to be specific to the paging audit app code?

Reply:
The definition is already present via AArch64Mmu.h so this can be removed